src/linker/utilities/spark_utils.py

```diff
@@ -15,7 +15,6 @@ def build_spark_cluster(
     session: "drmaa.Session",
```

Comment:
why do we still have drmaa stuff?

Reply:
the Spark cluster spinup isn't done through snakemake (actually it isn't done at all right now). I'm not sure if I'll be able to do it through snakemake but i should probably be able to free it from DRMAA either way. I wasn't going to mess with it until working on MIC-4830